src/cloud-functions.ts

```diff
@@ -149,7 +149,7 @@ export namespace Change {
   }
 
   /**
-   * @internal
+   * @hidden
```

Comment:
should we make these single lined?

Reply:
Yep: https://github.com/firebase/firebase-functions/pull/511/commits/53c3a1f88882c9bbbf6e0ef4c769d31701ca6854